codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/protocols/ClientProtocolLoader.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.rust.codegen.core.smithy.protocols
```

Comment:
This file should be renamed to `ProtocolLoader.kt`

Reply:
Good catch! Done in [Rename protocol loader in codegen-core from ClientProtocolLoader …](https://github.com/awslabs/smithy-rs/pull/1762/commits/0876167a0ed066c4db281e23ee9fa797ce98c191).